cmd/ack-generate/command/common.go

```diff
@@ -207,3 +212,67 @@ func getSDKVersionFromGoMod(goModPath string) (string, error) {
 	}
 	return "", fmt.Errorf("couldn't find %s in the go.mod require block", sdkModule)
 }
+
+// loadModelWithLatestAPIVersion finds the AWS SDK for a given service alias and
```

Comment:
this and the code below, considering (in a future PR if you agree with me) perhaps putting this into an importable package under `pkg/`